manage_config_work_threshold_page.php

```diff
@@ -410,6 +410,7 @@ function get_section_end() {
 get_section_begin_mcwt( lang_get( 'others' ) );
 get_capability_row( lang_get( 'view' ) . ' ' . lang_get( 'changelog_link' ), 'view_changelog_threshold' );
 get_capability_row( lang_get( 'view' ) . ' ' . lang_get( 'roadmap_link' ), 'roadmap_view_threshold' );
+get_capability_row( lang_get( 'view' ) . ' ' . lang_get( 'summary_link' ), 'view_summary_threshold' );
```

Comment:
Please add to Admin Guide Page_Descriptions.xml

Reply:
thanks, done (fixed original commit)